docs/notebooks/simulation/particle_stepper.ipynb

```diff
@@ -4,10 +4,7 @@
    "cell_type": "code",
```

Comment:
<p>Hm, this isn't the nicest way of getting out the times and positions from the save method. Maybe we should have a <code>results</code>  method or something on save routine that returns this stuff without the user having to know the names of variables like <code>t_all</code> ?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>t, x, v = save_routine.result </p><p> </p><p>do list -&gt; array conversion in that method </p>